Dockerfile.aggregation_job_creator

```diff
@@ -0,0 +1,14 @@
+FROM rust:1.60.0-alpine as builder
+RUN apk add libc-dev
+
+WORKDIR /src
+COPY Cargo.toml /src/Cargo.toml
+COPY Cargo.lock /src/Cargo.lock
+COPY janus_server /src/janus_server
+COPY monolithic_integration_test /src/monolithic_integration_test
+COPY test_util /src/test_util
+RUN --mount=type=cache,target=/usr/local/cargo/registry --mount=type=cache,target=/src/target cargo build --release --bin aggregation_job_creator && cp /src/target/release/aggregation_job_creator /aggregation_job_creator
```

Comment:
Is the only difference between the two dockerfiles `--bin aggregator` vs. `--bin aggregation_job_creator` and thenwhich path gets copied in from the builder image? If so, maybe we can use a single Dockerfile that takes a parameter for image name.

Reply:
Obviously this suggestion contradicts my other comment on `.github/workflows/ci-build.yml`. I'm comfortable with either a single or dual Dockerfile solution, so please move ahead with whichever one you prefer.

Reply:
I went with one parameterized file, nice idea! Build arguments can't be directly used in the ENTRYPOINT, unfortunately, so I decided to copy the argument into an environment variable (which becomes part of the image) and use that in a shell command. The alternate solution would be to rename the binary to something constant when copying it, and then put that common name in the ENTRYPOINT. I decided against that, as I think one PID plus a few megabytes of memory is worth the clarity of having argv[0] be the "real" binary name.